docs/weave/weave-api-spec/02-transaction-signatures.md

```diff
@@ -0,0 +1,60 @@
+---
+id: tx-sign-spec
+title: Transaction signitures 
+sidebar_label: Transaction signatures 
+---
+
+In this section, we will explain how to sign __transactions__.
+
+## Authentication and Authorization
+
+To execute desired actions and state changes on weave based blockchains, every transaction must be validated, authenticated and then authorized to perform the action. In the realm of blockchains this is achieved by using signatures.
+
+Weave uses `ed25519` signing algorithm to achieve this important feature. 
+After following the steps in [Weave transactions](transaction.md), created **Tx** must be signed. But first Weave *conditions* role in signing must be explained.
+
+[//]: # (TODO move readthedocs documentation to this project and give references here)
+
+Conditions format defined as `(extension, type, data)` in general. 
+
+Condition with __ed25519__ public key signature could be represented as `("sigs", "ed25519", <address>)`. 
+
+Here is the basic golang presentation of condition 
+
+`condition := sprintf("%s/%s/%X", extension, type, data)` where __condition__ looks something like `cond:sigs/ed25519/636f6e646974696f6e64617461`.
```

Comment:
This might be a preference and I might be wrong. When I read "here is ...", I am expecting a code example. Maybe it is better to use ``` for code formatting, so that the example takes the wole line.

There is no `sprintf`. Use `fmt.Sprintf`

```go
condition := fmt.Sprintf("%s/%s/%X", extension, type, data)
```